### DISCUSSION TITLE
My application for student benefit has a bug.

### DISCUSSION THREAD

**Discussion by Adi-Stanciu** [community/community#113675] [GitHub Education]
### Hi everyone,

So I wanted to upload my picture in order to get the student benefits. Once I ulpoaded it, it scanned it and said that I need to add a billing address and a 2 step authentification to my account first. I did so and after returning to upload my Student ID it said to me that the image has been already scanned and I can't scan it again, but I have no access to my benefits.

**Comment by samus-aran** [comment]
Hi @Adi-Stanciu 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
